docs/sources/guides/whats-new-in-v7-0.md

```diff
@@ -12,8 +12,184 @@ weight = -17
+
+### Support for backend plugins
+
+Grafana now officially supports backend plugins and the first type of plugin to be introduced is a backend component for data source plugins. You can optionally add a backend component to your data source plugin and implement the query logic there to automatically enable alerting in Grafana for your plugin. In the 7.0 release, we provide a Go SDK to build plugins and you can generate a plugin scaffold to help you get started using the [`@grafana/toolkit`](https://www.npmjs.com/package/@grafana/toolkit).
+
+Plugins can be monitored with the new metrics and health check capabilities. The new Resources capability means backend components can return non-time series data like JSON or static resources like images and opens up Grafana for new use cases.
+
+With this release, we are deprecating the unofficial first version of backend plugins which will be removed in a future release.
+
+## New tutorials
+
+To help you get started with Grafana, we’ve launched a brand new tutorials platform. We’ll continue to expand the platform with more tutorials, but here are some of the ones you can try out now:
+
+- [Grafana fundamentals](https://grafana.com/tutorials/grafana-fundamentals/)
+- [Create users and teams](https://grafana.com/tutorials/create-users-and-teams/)
+- [Build a panel plugin](https://grafana.com/tutorials/build-a-panel-plugin/)
+- [Build a data source plugin](https://grafana.com/tutorials/build-a-data-source-plugin/)
+
+## Rollup indicator for Metrictank queries
+
+{{< docs-imagebox img="/img/docs/v70/metrictank_rollup_metadata.png" max-width="800px" class="docs-image--right" caption="Metrictank rollup metadata" >}}
+
+Depending on the cardinality of the data and the time range MetricTank may return rolled up (aggregated) data. This can be as subtle as potentially only 1 or 2 graphs out of nine being rolled up. The new rollup indicator is visible in the panel title and you can also inspect extensive metadata and stats about the Metrictank query result and its rollups.
+
+<div class="clearfix"></div>
+
+## Breaking change - PhantomJS removed
+
+[PhantomJS](https://phantomjs.org/), have been used for rendering images of dashboards and panels and have been included with Grafana since Grafana v2.0. Since then we’ve had a lot of related bugs and security related issues, mainly due to the fact that PhantomJS have struggled with supporting modern web technologies. Throughout the years, maintaining PhantomJS support in Grafana has been a nightmare. Removing support for PhantomJS has been a high priority for the Grafana project and got stressed even more when the PhantomJS maintainer in March 2018 [announced](https://github.com/ariya/phantomjs/issues/15344) the end of the project.
```

Comment:
```suggestion
[PhantomJS](https://phantomjs.org/) was used for rendering images of dashboards and panels and has been included with Grafana since Grafana v2.0. Since then, we’ve had a lot of related bugs and security-related issues, mainly due to the fact that PhantomJS have struggled with supporting modern web technologies. Throughout the years, maintaining PhantomJS support in Grafana has been a nightmare. Removing support for PhantomJS has been a high priority for the Grafana project and got stressed even more when the PhantomJS maintainer in March 2018 [announced](https://github.com/ariya/phantomjs/issues/15344) the end of the project.
```

Reply:
This is not necessary. We don't need to to go on about how hard it was or the history.

We removed PhantomJS. We replaced it with this. Here are the changes.